ironblogger/static/style.css

```diff
@@ -81,6 +81,12 @@ section {
   overflow:hidden;
 }
 
+section > p > a {
```

Comment:
You could also just do git commit --amend; git push -f. The main thing is it seems silly to open a second issue.